mergebot_poller.py

```diff
@@ -105,7 +105,7 @@ def search_github_pr(self, pull):
         """
         # Load comments for each pull request
         comments = pull.fetch_comments()
-        if comments == []:
+        if not comments:
             print_flush('No comments. Moving on.')
```

Comment:
This is covered above in the file; since this is single-threaded, all messages regarding a pull request will be between a <PR #x> ... </PR #x>.